### PR TITLE
Add FOSDEM 2020 post

### PR DESCRIPTION
**Pull request by nikolas**

### CODE REVIEW

File: content/articles/fosdem-2020.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+socmediaimg: ""
+hiliteimg: ""
+poster_sourceurl: ""
+poster_source: ""
+bookendanimal: ""
+---
+
+I attended [FOSDEM](https://fosdem.org/2020/) in Brussels on February 1st and 2nd, last
+weekend. It was pretty overwhelming, the amount of people throughout
+the [ULB](https://www.ulb.be/en) campus. The number of attendees is around 8,000, and at
```
Comment (sdreher) [inline content/articles/fosdem-2020.md]: Sentence is a bit awkward. Maybe combine with the next one?

File: content/articles/fosdem-2020.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+Here's an overview of some of the talks I took notes on, with links to
+the recordings.
+
+# [Over Twenty Years Of Automation](https://fosdem.org/2020/schedule/event/automation/) - James Shubin
+
+The creator of [mgmtconfig](https://github.com/purpleidea/mgmt)
+describes the history of configuration management.
+He gives demos for many different systems,
+including a simple shell scripting + ssh combo,
+puppet, chef, ansible, docker, and others. He
```
Comment (sdreher) [inline content/articles/fosdem-2020.md]: capitalize the software packages here?

File: content/articles/fosdem-2020.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+This talk was completely packed, in the main J auditorium. I'm not
+sure why. The author of curl describes making a new, faster HTTP
+protocol based on UDP instead of TCP.
+
+<hr />
+
+I saw some interesting talks, but there's at least as much info to dig
+through in the video recordings for all the talks, here:
+https://video.fosdem.org/2020/ (These are organized by room
+number.. the title/speaker info for each talk can be found
```
Comment (sdreher) [inline content/articles/fosdem-2020.md]: ...

File: content/articles/fosdem-2020.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+a more self-contained way: [systemd-homed](https://www.phoronix.com/scan.php?page=news_item&px=systemd-homed). The first thought that comes
+to my mind is... why??? I'll just have to watch the talk.
+
+## [Python](https://fosdem.org/2020/schedule/track/python/) and [JavaScript](https://fosdem.org/2020/schedule/track/javascript/) dev rooms
+
+<hr />
+
+And here are just some random photos from around the conference.
+
+FOSDEM's special beer, with a Belgian beer Cuvée des Trolls.
```
Comment (sdreher) [inline content/articles/fosdem-2020.md]: :yum:

File: content/articles/fosdem-2020.md (hunk 1 above)
Comment (nikolas) [reply content/articles/fosdem-2020.md]: I re-worded this to use less passive voice.

**Pull request merged**